src/index.js

```diff
@@ -3,9 +3,16 @@ import RandomInput from './random-input';
 import Translate from './translate';
 import ValuesInput from './values-input';
 
-export {
+const brainCT = {
```

Comment:
What if we put RandomInput, Translate, and ValuesInput as statics on BrainCT?

Reply:
yeah, I messed up the `export` altogether, I'll throw in a smaller PR to fix this